geomesa-raster/src/main/scala/org/locationtech/geomesa/raster/data/Raster.scala

```diff
@@ -38,9 +39,31 @@ case class Raster(chunk: RenderedImage, metadata: DecodedIndex, resolution: Doub
 
   lazy val serializedChunk = RasterUtils.imageSerialize(chunk)
 
+  def encodeValue = RasterUtils.imageSerialize(chunk)
+
+  def encodeToBytes(): Array[Byte] = {
+    val chunkBytes = RasterUtils.imageSerialize(chunk)
+    val metaDataBytes = metadata.toBytes
+
+    ByteBuffer.allocate(4).putInt(chunkBytes.length).array() ++ chunkBytes ++
+      ByteBuffer.allocate(4).putInt(metaDataBytes.length).array() ++ metaDataBytes ++
+      ByteBuffer.allocate(8).putDouble(resolution).array()
```

Comment:
Same comment as above.  During a later ticket, we may want to explore Avro encoding for this info.